tastebuddies/__init__.py

```diff
@@ -0,0 +1,76 @@
+from pyramid.config import Configurator
+from sqlalchemy import engine_from_config
+
+import os
+
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+from cryptacular.bcrypt import BCRYPTPasswordManager
+
+from .models import (
+    DBSession,
+    Base,
+)
+
+from security import groupfinder, Root, UserFactory
+from pyramid.view import forbidden_view_config
+
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application.
+    """
+
+    @forbidden_view_config(renderer='templates/forbidden.jinja2')
+    def forbidden(request):
+        return {}
```

Comment:
Might be better to put this into the `views.py` file instead of inlining it here.